modules/swagger-codegen/src/main/resources/python/model.mustache

```diff
@@ -73,6 +75,10 @@ class {{classname}}(object):
 {{/required}}
 {{/vars}}
 
+{{#parent}}
+        super().__init__({{#parentModel.vars}}{{name}}{{#hasMore}}, {{/hasMore}}{{/parentModel.vars}})  # noqa: E501
+
+{{/parent}}
```

Comment:
The parent constructor is called with the values passed in to the constructor of the child

Reply:
I believe this should go up to the first line below the __init__ function so that calling the parent constructor does not wipe out any properties that were overridden by the current child class.

Reply:
I believe I had reasoned that it didn't matter much because the code was using `vars` for the child class properties and `parentModel.vars` for properties on the parent. Since the child class never worked with `parentModel.vars` directly it would be fine. However, the discriminator (and any other code that is added in the future) might not realize this, so I changed the order of events